packages/millicast-viewer-demo/src/viewer.js

```diff
@@ -242,27 +247,63 @@ document.addEventListener("DOMContentLoaded", () => {
     false
   );
 
+  registerRequestVideoFrameCallback(video);
+
   int = setInterval(() => {
     clearInterval(int);
   }, 2000);
   subscribe();
 });
 
+function registerRequestVideoFrameCallback(videoElem) {
+  const captionElem = document.getElementById('caption');
+  let curCC = null
+  if ("requestVideoFrameCallback" in HTMLVideoElement.prototype) {
+    const processor = (now, { rtpTimestamp }) => {
+      const ts = rtpTimestamp / 1000;
```

Comment:
question: @vincentsong would this work with any Millicast RTS stream or does it need to have timestamp metadata enabled? When I was looking at using this in the hybrid player, I was not getting the accurate timestamps I was hoping for. I was looking for the `captureTime` property, but it was not present in the Millicast RTS stream's frame data. 

Also, does this work in Firefox? I recall it not having full support for the video frame callback.